docs/install_linux.md

```diff
@@ -167,6 +167,19 @@ openSUSE Tumbleweed users can install from the [official distribution repo](http
 sudo zypper in gh
 ```
 
+### Alpine Linux
+
+Alpine Linux users can install from the `testing` repository, under `edge`:
```

Comment:
How long is the package going to stay under "testing"? Is it possible that it will get promoted to the default repository in the near future and thus installable via `apk add github-cli`?

I'm asking because maybe we can hold off offering this documentation until the package is "official"

Reply:
Sure! I'll try to move it to `community` repo and possibly backport to atleast `v3.14` soon.

Reply:
Thanks. I'll close this PR for now, but please open a similar PR once the package lands in community and is easily available.